Buffers/BufferIn.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import numpy as np
+testData = np.array([[1,2,3],[4,5,6]],dtype=np.uint8)
+def createBufferIn(data):
+    buffer = np.unpackbits(data).flatten()
+    print(buffer)
+    countIn = 8 #Na przykład 8, wielkość ramki bez nagłówków
+    offsetIn = 0
+    for offsetIn in range (0, len(buffer)-1, countIn): #Pętla opróżniająca cały buffer
+            dataIn = np.frombuffer(buffer, dtype='uint8', count=countIn,offset=offsetIn)
+            if len(dataIn) % 2 != 0:
+                raise Exception('Data is uneven!')
+                returnVal = []
+                return returnVal
+#frame=headers+datain
+#funkcja wysyłająca, zamiast tego teraz wyświetlanie
+            print("Zawartosc ramki to:",dataIn)
+            offsetIn = offsetIn + countIn
+
+createBufferIn(testData)
+
```

Comment:
ogólnie to ja Ci tego w takiej formie na pewno nie klepnę:
1. Wytłumacz mi po co jest Twój kod i jak ma działać
2. Jeżeli to jest plik zawierający test to zmień nazwę pliku, jeżeli nie to nie testuj w tym samym pliku, bo ogółem to my będziemy to potem wykorzystywać, a taka forma jest niedopuszczalna
3. Staraj się unikać zbędnych pętli i if'ów chyba, że już nie da się uniknąć, wtedy już trudno